tls/s2n_cipher_preferences.h

```diff
@@ -21,7 +21,8 @@
 
 
 #define S2N_ECC_EXTENSION_ENABLED 0x01
-#define S2N_SIKE_EXTENSION_ENABLED 0x02
+#define S2N_BIKE_EXTENSION_ENABLED 0x02
```

Comment:
Are we headed down a dangerous path using an 8-bit bit mask to signal extensions?